core/types/calls.cc

```diff
@@ -4238,6 +4264,16 @@ const vector<Intrinsic> intrinsics{
 
     {Symbols::Module(), Intrinsic::Kind::Instance, Names::tripleEq(), &Module_tripleEq},
     {Symbols::T_Enum(), Intrinsic::Kind::Instance, Names::tripleEq(), &T_Enum_tripleEq},
+
+    {Symbols::T_Array(), Intrinsic::Kind::Singleton, Names::tripleEq(), &GenericForwarder_tripleEq},
```

Comment:
It's too bad this is another list of `T` generics that has to be kept up with other places that we care about (although it's not the end of the world if this list isn't updated along with the others).